AirCasting/SessionViews/ShareSession/ShareSessionStreamOptionViewModel.swift

```diff
@@ -0,0 +1,20 @@
+// Created by Lunar on 16/12/2021.
+//
+
+import Foundation
+
+struct ShareSessionStreamOptionViewModel {
+    let id: Int
+    let title: String
+    let streamName: String
+    var isSelected: Bool
+    let isEnabled: Bool
+
+    mutating func toggleSelection() {
+        isSelected.toggle()
+    }
+
+    mutating func changeSelection(newSelected: Bool) {
+        isSelected = newSelected
+    }
+}
```

Comment:
I have no hard opinion here, and it would be best to ask @pawelgil. Assuming I know smth, I would like to know why it is better than the class? We are used to viewModels being class I think. Can this one be class too? I think it is a bit rule-breaking, as long as the struct is immutable we should leave it like this, but as I mentioned above I am not sure. I am going to read about it and increase my knowledge level

Reply:
Ah, you are right. It's just by mistake, I also think it should be a class. Thanks for catching that :)

Reply:
Hm, now I see that I took your `struct DeleteSessionOptionViewModel` and adapted it for sharing purposes. So I'm not sure if it should be a struct or a class. In your opinion, should I change `DeleteSessionOptionViewModel` to class as well?